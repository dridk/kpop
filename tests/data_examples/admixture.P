0.761150 0.676950 0.418566 0.385930
0.268861 0.999990 0.923561 0.821896
0.933572 0.567519 0.969779 0.101716
0.565509 0.369606 0.209565 0.907119
0.956751 0.000010 0.190604 0.544315
0.819399 0.999990 0.989610 0.324065
0.821331 0.394517 0.431529 0.481860
0.533855 0.999990 0.975746 0.234704
0.871894 0.999990 0.999990 0.764763
0.926285 0.728185 0.999990 0.655814
0.310846 0.474349 0.755065 0.999990
0.864537 0.279168 0.000010 0.849507
0.633657 0.913971 0.895123 0.687190
0.808776 0.132080 0.684817 0.985157
0.742185 0.758109 0.477272 0.340943
0.742639 0.456978 0.783319 0.601332
0.480355 0.144786 0.154362 0.898518
0.029918 0.815759 0.854887 0.665650
0.878273 0.303987 0.000010 0.948228
0.715206 0.664510 0.721809 0.746507
0.763233 0.363733 0.727788 0.205701
0.895555 0.999990 0.822189 0.425872
0.839643 0.766585 0.469136 0.430477
0.694402 0.312999 0.086237 0.896308
0.823860 0.999990 0.804637 0.636453
0.469575 0.508983 0.323656 0.999990
0.476529 0.551857 0.314529 0.770452
0.658119 0.546138 0.541469 0.780541
0.201502 0.920158 0.835788 0.606522
0.296981 0.374171 0.644532 0.757127
0.905238 0.291181 0.581960 0.731305
0.824692 0.779980 0.999990 0.147463
0.950926 0.084536 0.103121 0.968980
0.844517 0.539981 0.588398 0.213510
0.405088 0.714888 0.686396 0.571564
0.629139 0.920456 0.899248 0.148442
0.835221 0.825945 0.762574 0.520719
0.000010 0.822375 0.914873 0.999990
0.409831 0.999990 0.958849 0.741891
0.975559 0.000010 0.000010 0.784663
0.244833 0.841648 0.777477 0.612126
0.396622 0.999990 0.519258 0.772062
0.623136 0.124255 0.092155 0.999990
0.177838 0.999990 0.951344 0.693133
0.962225 0.000010 0.005163 0.591004
0.787204 0.847542 0.821146 0.119612
0.658358 0.999990 0.999990 0.330915
0.744104 0.892147 0.886022 0.454637
0.271993 0.999990 0.884232 0.537820
0.415778 0.484653 0.597814 0.794307
0.271077 0.498998 0.731689 0.690964
0.668858 0.669588 0.702932 0.180365
0.898029 0.864308 0.999990 0.999990
0.911470 0.395833 0.208494 0.528629
0.438909 0.999990 0.299869 0.701121
0.914309 0.800798 0.966184 0.771209
0.572662 0.740119 0.710208 0.984311
0.181091 0.999990 0.834244 0.825139
0.496901 0.533951 0.901401 0.399454
0.699622 0.927894 0.913012 0.120834
0.768780 0.299789 0.526096 0.925427
0.652764 0.504758 0.654133 0.520744
0.000010 0.999990 0.785316 0.876820
0.559682 0.613844 0.497747 0.841726
0.973775 0.588244 0.317473 0.643953
0.787184 0.821028 0.999990 0.876980
0.894132 0.490539 0.579552 0.552570
0.482062 0.575047 0.253058 0.674701
0.655577 0.999990 0.953348 0.278617
0.704880 0.999990 0.977603 0.827239
0.789833 0.217369 0.760839 0.608989
0.551335 0.999990 0.999990 0.937815
0.819654 0.999990 0.933963 0.560757
0.788644 0.597484 0.664999 0.433221
0.793710 0.707731 0.888769 0.047926
0.237214 0.918448 0.902299 0.401272
0.962974 0.466984 0.698413 0.819758
0.276058 0.999990 0.581645 0.998936
0.106477 0.613495 0.404353 0.952589
0.999990 0.981760 0.745640 0.820636
0.660622 0.999990 0.999990 0.183360
0.876175 0.999990 0.869165 0.572743
0.377982 0.533051 0.816783 0.684558
0.999990 0.914179 0.799538 0.263597
0.518660 0.121531 0.000010 0.999990
0.860786 0.939941 0.551692 0.000010
0.399682 0.729512 0.999990 0.280686
0.552842 0.999990 0.644566 0.299887
0.783727 0.541181 0.449854 0.852036
0.446670 0.825719 0.980237 0.828757
0.877543 0.999990 0.999990 0.002429
0.605773 0.999990 0.999990 0.562944
0.886875 0.626892 0.782391 0.340846
0.665942 0.297404 0.325591 0.556883
0.852312 0.243766 0.000010 0.708072
0.837004 0.575167 0.819636 0.979783
0.444045 0.999990 0.999990 0.645196
0.847809 0.523615 0.676398 0.999990
0.052694 0.999990 0.856673 0.612627
0.147638 0.999990 0.702757 0.999990
0.867379 0.192700 0.090086 0.622126
0.716231 0.866633 0.999990 0.213480
0.999990 0.297320 0.999990 0.000010
0.478624 0.916739 0.930576 0.430633
0.999990 0.398852 0.538717 0.804513
0.956793 0.999990 0.999990 0.678530
0.905137 0.999990 0.999990 0.260896
0.965178 0.929088 0.999990 0.218692
0.999990 0.330440 0.884435 0.491589
0.754164 0.964896 0.919528 0.045109
0.884011 0.342688 0.455992 0.306270
0.683982 0.999990 0.720530 0.106586
0.209982 0.234550 0.623126 0.918511
0.368766 0.859953 0.999990 0.964240
0.871206 0.264339 0.000010 0.640752
0.308833 0.224408 0.223022 0.999990
0.584789 0.999990 0.970661 0.194029
0.822739 0.999990 0.804771 0.999990
0.984558 0.965032 0.912648 0.628780
0.849338 0.508092 0.999990 0.499592
0.765055 0.704020 0.618468 0.635215
0.120403 0.772071 0.853487 0.598897
0.785118 0.684951 0.965457 0.858539
0.194896 0.598141 0.600774 0.735407
0.971848 0.999989 0.999990 0.000010
0.957838 0.999990 0.683566 0.869416
0.999990 0.660213 0.179728 0.789939
0.716893 0.000010 0.490898 0.756187
0.345201 0.886989 0.999990 0.689139
0.706507 0.473834 0.477303 0.629879
0.854334 0.307029 0.212669 0.603896
0.419041 0.465598 0.961352 0.699128
0.906240 0.178920 0.762575 0.308758
0.843412 0.375588 0.469403 0.404109
0.869413 0.280350 0.461817 0.570364
0.205981 0.755965 0.817545 0.999990
0.999990 0.711715 0.137969 0.380411
0.295408 0.999990 0.918277 0.306534
0.786798 0.624404 0.690748 0.845294
0.999990 0.849474 0.722487 0.217165
0.637881 0.226471 0.381758 0.936385
0.626249 0.930765 0.999990 0.000010
0.821444 0.834629 0.999990 0.124819
0.609644 0.874843 0.967642 0.917954
0.879374 0.986660 0.959262 0.785646
0.398430 0.724517 0.999990 0.231690
0.000010 0.815961 0.754477 0.999990
0.693516 0.801553 0.999990 0.690604
0.112711 0.999990 0.968812 0.417285
0.106619 0.851173 0.742354 0.927104
0.999990 0.000010 0.331608 0.820577
0.392453 0.999990 0.997820 0.725638
0.478984 0.999990 0.855739 0.816684
0.467810 0.788279 0.846205 0.374808
0.293094 0.677645 0.589263 0.999990
0.151663 0.999990 0.909753 0.796051
0.582181 0.769714 0.919230 0.163764
0.531690 0.989147 0.999990 0.131974
0.808568 0.907856 0.913286 0.858755
0.872920 0.858067 0.628291 0.119482
0.592285 0.536591 0.564294 0.436923
0.353793 0.281985 0.510992 0.733996
0.999990 0.643190 0.210956 0.993738
0.999990 0.003279 0.148700 0.380871
0.732167 0.311971 0.597199 0.350350
0.999990 0.284762 0.014082 0.999990
0.372998 0.999990 0.970967 0.241720
0.410909 0.999990 0.638251 0.331593
0.965983 0.926265 0.807286 0.238298
0.956251 0.000010 0.000010 0.757282
0.520267 0.716851 0.902901 0.960829
0.366152 0.845954 0.921839 0.737446
0.340553 0.893747 0.495668 0.966614
0.366961 0.648946 0.403491 0.670166
0.953814 0.377156 0.744662 0.942675
0.999990 0.957377 0.229221 0.647963
0.824827 0.794877 0.530701 0.833053
0.174726 0.924953 0.794280 0.744776
0.296890 0.301763 0.750754 0.731995
0.708328 0.647143 0.474685 0.618783
0.529494 0.469717 0.704279 0.785139
0.054748 0.999990 0.654973 0.730545
0.344276 0.999990 0.939359 0.450157
0.454219 0.349565 0.146007 0.865087
0.829007 0.826418 0.725526 0.970480
0.693566 0.323401 0.458585 0.513971
0.577634 0.698241 0.806082 0.225913
0.999990 0.609442 0.093011 0.556760
0.811833 0.206949 0.582521 0.868063
0.999990 0.754013 0.996428 0.612100
0.924817 0.102326 0.000010 0.672983
0.961313 0.729005 0.927088 0.784856
0.069496 0.818556 0.676141 0.748782
0.942774 0.606330 0.400678 0.689288
0.999990 0.733134 0.817364 0.364961
0.420723 0.750492 0.886221 0.878080
0.068084 0.969168 0.880906 0.999990
0.979282 0.000010 0.055936 0.837678
0.992290 0.999990 0.913257 0.756749
0.624289 0.840800 0.426518 0.813833