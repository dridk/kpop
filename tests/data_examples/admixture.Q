0.000010 0.000010 0.000010 0.999970
0.000010 0.000010 0.000010 0.999970
0.044000 0.055150 0.000010 0.900841
0.000010 0.000010 0.050735 0.949245
0.000010 0.000010 0.086763 0.913217
0.000010 0.000010 0.051575 0.948405
0.000010 0.062394 0.000010 0.937586
0.000010 0.000010 0.042912 0.957068
0.000010 0.000010 0.000010 0.999970
0.000010 0.064253 0.000010 0.935727
0.036504 0.000010 0.000010 0.963476
0.000010 0.027710 0.000010 0.972270
0.040247 0.069244 0.042733 0.847776
0.000010 0.000010 0.000010 0.999970
0.000010 0.000010 0.040059 0.959921
0.000010 0.000010 0.000010 0.999970
0.000010 0.000010 0.029978 0.970002
0.000010 0.049350 0.000010 0.950630
0.014895 0.000010 0.070735 0.914359
0.059783 0.017044 0.000010 0.923163
0.000010 0.097438 0.000010 0.902542
0.000010 0.038042 0.000010 0.961938
0.000014 0.066387 0.000010 0.933589
0.000010 0.000010 0.000010 0.999970
0.000010 0.000010 0.101196 0.898784
0.000010 0.050680 0.000010 0.949300
0.000010 0.000010 0.000010 0.999970
0.000010 0.000010 0.000010 0.999970
0.011358 0.000010 0.057904 0.930728
0.000010 0.000010 0.058676 0.941304
0.000010 0.000010 0.000010 0.999970
0.013449 0.000010 0.046467 0.940075
0.062117 0.000010 0.000010 0.937863
0.000010 0.000010 0.036330 0.963650
0.000010 0.000010 0.017455 0.982525
0.000010 0.000010 0.000010 0.999970
0.000010 0.000010 0.000010 0.999970
0.061004 0.000010 0.024351 0.914635
0.000010 0.000010 0.000010 0.999970
0.000010 0.000010 0.000010 0.999970
0.000010 0.005632 0.000010 0.994348
0.036089 0.000010 0.000010 0.963891
0.000010 0.051773 0.000010 0.948207
0.043962 0.030020 0.000010 0.926008
0.045456 0.028858 0.000010 0.925676
0.000010 0.000010 0.000010 0.999970
0.021613 0.000010 0.044120 0.934257
0.000010 0.000010 0.000010 0.999970
0.000010 0.000010 0.036211 0.963769
0.000010 0.000010 0.070415 0.929565
0.000010 0.038154 0.000010 0.961826
0.029156 0.063318 0.000010 0.907516
0.003204 0.000010 0.000010 0.996776
0.000010 0.000010 0.000010 0.999970
0.000010 0.013826 0.000010 0.986154
0.041831 0.000010 0.027840 0.930319
0.030104 0.034970 0.000010 0.934917
0.098345 0.016936 0.000010 0.884709
0.044283 0.000010 0.005168 0.950539
0.000010 0.000010 0.000010 0.999970
0.041537 0.023069 0.000010 0.935384
0.000010 0.000010 0.095770 0.904210
0.043182 0.000010 0.000010 0.956798
0.000010 0.000010 0.003910 0.996070
0.000010 0.000010 0.037202 0.962778
0.000010 0.000010 0.028327 0.971653
0.059741 0.000010 0.000010 0.940239
0.013862 0.138042 0.000010 0.848086
0.000010 0.000010 0.027582 0.972398
0.000010 0.001983 0.000010 0.997997
0.000010 0.000010 0.000010 0.999970
0.000010 0.000010 0.000010 0.999970
0.072590 0.000010 0.000010 0.927390
0.000010 0.011436 0.000010 0.988544
0.000010 0.039296 0.000010 0.960684
0.000010 0.000010 0.000010 0.999970
0.000010 0.030008 0.000010 0.969972
0.048019 0.000010 0.000010 0.951961
0.033144 0.044488 0.000010 0.922358
0.000010 0.000010 0.000010 0.999970
0.000010 0.000010 0.000010 0.999970
0.000010 0.000010 0.000010 0.999970
0.000010 0.000010 0.000010 0.999970
0.055072 0.013945 0.000010 0.930973
0.011793 0.042909 0.000010 0.945288
0.022182 0.000010 0.000010 0.977798
0.000010 0.000010 0.000010 0.999970
0.000010 0.000010 0.022489 0.977491
0.032655 0.042455 0.000010 0.924880
0.010177 0.000010 0.033974 0.955838
0.004857 0.000010 0.004082 0.991050
0.000010 0.000010 0.000010 0.999970
0.000010 0.096639 0.000010 0.903341
0.000010 0.000010 0.000010 0.999970
0.000010 0.036479 0.000010 0.963501
0.052233 0.000010 0.000010 0.947747
0.000010 0.012553 0.000010 0.987427
0.000010 0.000010 0.037522 0.962458
0.000010 0.041668 0.000010 0.958312
0.018890 0.000010 0.022196 0.958904
0.000010 0.245293 0.670306 0.084391
0.000010 0.466722 0.533258 0.000010
0.013079 0.000010 0.986901 0.000010
0.055926 0.152447 0.791617 0.000010
0.012982 0.392444 0.594564 0.000010
0.044617 0.635391 0.319982 0.000010
0.000010 0.180679 0.780520 0.038791
0.050894 0.202944 0.719431 0.026730
0.000010 0.516420 0.483560 0.000010
0.000010 0.301167 0.650285 0.048538
0.000010 0.122505 0.788356 0.089129
0.012125 0.195232 0.792632 0.000010
0.000010 0.352171 0.647809 0.000010
0.043115 0.288929 0.667947 0.000010
0.000010 0.314437 0.685543 0.000010
0.000010 0.000010 0.999970 0.000010
0.000010 0.263835 0.727058 0.009097
0.026155 0.367417 0.577109 0.029319
0.001983 0.236138 0.761869 0.000010
0.020011 0.146898 0.833080 0.000010
0.106765 0.123235 0.718264 0.051736
0.027137 0.174184 0.705766 0.092913
0.000010 0.245957 0.754023 0.000010
0.000010 0.594051 0.405929 0.000010
0.092105 0.224862 0.683023 0.000010
0.024263 0.280421 0.695306 0.000010
0.005542 0.338249 0.641507 0.014701
0.030867 0.116721 0.852402 0.000010
0.068374 0.571109 0.360507 0.000010
0.000010 0.603847 0.363258 0.032885
0.000010 0.515971 0.484009 0.000010
0.000010 0.260078 0.716637 0.023275
0.041103 0.145964 0.770849 0.042084
0.000010 0.386693 0.613287 0.000010
0.064496 0.162141 0.773353 0.000010
0.000010 0.955319 0.000010 0.044661
0.000010 0.563476 0.400434 0.036080
0.000010 0.000010 0.999970 0.000010
0.002830 0.382316 0.614844 0.000010
0.017542 0.089251 0.818530 0.074678
0.025551 0.790745 0.153949 0.029756
0.034754 0.346405 0.618831 0.000010
0.000010 0.000010 0.999970 0.000010
0.000010 0.033864 0.929440 0.036685
0.026119 0.459306 0.495547 0.019028
0.057603 0.466457 0.450701 0.025239
0.000010 0.000010 0.999970 0.000010
0.000010 0.268613 0.622823 0.108554
0.066593 0.000010 0.933387 0.000010
0.076164 0.147260 0.776566 0.000010
0.000010 0.300961 0.699019 0.000010
0.000010 0.000010 0.970890 0.029090
0.000010 0.244151 0.755829 0.000010
0.045463 0.318705 0.618168 0.017665
0.000010 0.162849 0.821597 0.015543
0.000010 0.153639 0.792622 0.053729
0.000010 0.232191 0.767789 0.000010
0.000010 0.371580 0.628400 0.000010
0.008084 0.000010 0.946975 0.044931
0.000010 0.244475 0.700877 0.054639
0.060373 0.532695 0.406922 0.000010
0.000010 0.146728 0.804818 0.048444
0.011312 0.313671 0.675007 0.000010
0.019714 0.219714 0.760561 0.000010
0.065085 0.000010 0.907650 0.027255
0.026123 0.232759 0.712078 0.029040
0.003700 0.462141 0.534149 0.000010
0.000010 0.530141 0.469839 0.000010
0.057905 0.000010 0.911108 0.030977
0.000010 0.394425 0.597196 0.008369
0.030764 0.337603 0.599283 0.032350
0.018748 0.451375 0.480489 0.049388
0.027581 0.251582 0.720827 0.000010
0.054428 0.479689 0.465873 0.000010
0.000852 0.160529 0.838609 0.000010
0.000010 0.462680 0.537300 0.000010
0.017908 0.500313 0.481769 0.000010
0.000010 0.485416 0.514564 0.000010
0.007768 0.319724 0.672498 0.000010
0.000010 0.000010 0.883799 0.116181
0.008298 0.574440 0.366288 0.050974
0.121642 0.361718 0.501115 0.015524
0.011540 0.221638 0.746007 0.020815
0.002015 0.218651 0.779324 0.000010
0.016371 0.014404 0.940552 0.028672
0.011607 0.577640 0.388387 0.022367
0.000010 0.000010 0.999970 0.000010
0.050938 0.096763 0.823875 0.028423
0.000010 0.282763 0.717217 0.000010
0.014616 0.252496 0.704419 0.028469
0.000010 0.359449 0.634104 0.006437
0.035891 0.108325 0.849473 0.006311
0.099861 0.279376 0.583907 0.036856
0.034990 0.145305 0.819696 0.000010
0.005688 0.132489 0.855760 0.006063
0.000010 0.999970 0.000010 0.000010
0.028592 0.355172 0.616226 0.000010
0.000010 0.545570 0.454410 0.000010
0.018703 0.268376 0.712910 0.000010
0.025033 0.616978 0.357979 0.000010
0.920136 0.024747 0.040874 0.014243
0.959117 0.000010 0.000010 0.040863
0.961680 0.017397 0.000010 0.020913
0.995598 0.004382 0.000010 0.000010
0.908856 0.061167 0.000010 0.029967
0.915500 0.084480 0.000010 0.000010
0.926281 0.000010 0.000010 0.073699
0.999970 0.000010 0.000010 0.000010
0.999242 0.000010 0.000010 0.000738
0.941106 0.058874 0.000010 0.000010
0.937277 0.000010 0.062703 0.000010
0.956627 0.005854 0.000010 0.037509
0.999970 0.000010 0.000010 0.000010
0.999970 0.000010 0.000010 0.000010
0.908044 0.091936 0.000010 0.000010
0.947736 0.000010 0.000010 0.052244
0.951574 0.000010 0.048406 0.000010
0.974367 0.025613 0.000010 0.000010
0.994539 0.005441 0.000010 0.000010
0.999629 0.000351 0.000010 0.000010
0.950901 0.049079 0.000010 0.000010
0.999970 0.000010 0.000010 0.000010
0.957230 0.034553 0.000010 0.008207
0.935851 0.064129 0.000010 0.000010
0.936650 0.000010 0.024333 0.039006
0.999589 0.000391 0.000010 0.000010
0.975446 0.000010 0.000010 0.024534
0.912432 0.015413 0.072145 0.000010
0.920415 0.062701 0.000010 0.016874
0.885616 0.000010 0.000010 0.114364
0.969144 0.000010 0.000010 0.030836
0.957666 0.042314 0.000010 0.000010
0.948692 0.000010 0.051288 0.000010
0.958149 0.000010 0.041831 0.000010
0.945901 0.000010 0.054079 0.000010
0.999970 0.000010 0.000010 0.000010
0.922092 0.016868 0.000010 0.061030
0.938764 0.061216 0.000010 0.000010
0.972273 0.027707 0.000010 0.000010
0.931941 0.000010 0.033390 0.034659
0.943711 0.049611 0.000010 0.006667
0.934102 0.000010 0.000010 0.065878
0.951510 0.000010 0.013025 0.035456
0.999970 0.000010 0.000010 0.000010
0.977636 0.011878 0.000010 0.010476
0.961205 0.000010 0.000010 0.038775
0.988443 0.011537 0.000010 0.000010
0.947499 0.000010 0.000010 0.052481
0.984920 0.000010 0.015060 0.000010
0.979447 0.000010 0.007961 0.012582
0.918538 0.000010 0.063127 0.018326
0.999970 0.000010 0.000010 0.000010
0.910193 0.014014 0.000010 0.075783
0.999970 0.000010 0.000010 0.000010
0.957401 0.042579 0.000010 0.000010
0.953816 0.000010 0.000010 0.046164
0.942126 0.000010 0.005594 0.052270
0.919282 0.028918 0.000010 0.051790
0.985477 0.014503 0.000010 0.000010
0.950708 0.015485 0.000010 0.033797
0.999970 0.000010 0.000010 0.000010
0.999970 0.000010 0.000010 0.000010
0.986794 0.000010 0.000010 0.013186
0.987829 0.000010 0.000010 0.012151
0.971189 0.028791 0.000010 0.000010
0.963653 0.000010 0.011816 0.024521
0.941066 0.058914 0.000010 0.000010
0.999070 0.000010 0.000010 0.000910
0.959292 0.000010 0.003588 0.037110
0.964215 0.000010 0.000010 0.035765
0.942635 0.000010 0.004082 0.053272
0.947048 0.052932 0.000010 0.000010
0.999970 0.000010 0.000010 0.000010
0.956513 0.000010 0.000010 0.043467
0.943139 0.051601 0.000010 0.005250
0.970436 0.000010 0.000010 0.029544
0.953545 0.046435 0.000010 0.000010
0.947364 0.036856 0.000010 0.015770
0.937288 0.062692 0.000010 0.000010
0.976761 0.000010 0.023219 0.000010
0.979293 0.000010 0.020687 0.000010
0.969464 0.000010 0.000010 0.030516
0.917565 0.000010 0.082415 0.000010
0.966548 0.000010 0.033432 0.000010
0.990514 0.009466 0.000010 0.000010
0.936341 0.000010 0.063639 0.000010
0.953611 0.000010 0.031289 0.015090
0.946499 0.000010 0.000010 0.053481
0.973063 0.000010 0.026917 0.000010
0.976149 0.000010 0.000010 0.023831
0.906134 0.000010 0.000010 0.093846
0.928198 0.054804 0.000010 0.016988
0.981336 0.000010 0.000010 0.018644
0.935786 0.064194 0.000010 0.000010
0.999970 0.000010 0.000010 0.000010
0.990832 0.009148 0.000010 0.000010
0.941049 0.000010 0.058931 0.000010
0.923296 0.000010 0.005947 0.070747
0.967487 0.032493 0.000010 0.000010
0.948101 0.000010 0.023528 0.028361