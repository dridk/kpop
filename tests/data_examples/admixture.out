****                   ADMIXTURE Version 1.3.0                  ****
****                    Copyright 2008-2015                     ****
****           David Alexander, Suyash Shringarpure,            ****
****                John  Novembre, Ken Lange                   ****
****                                                            ****
****                 Please cite our paper!                     ****
****   Information at www.genetics.ucla.edu/software/admixture  ****

Random seed: 43
Point estimation method: Block relaxation algorithm
Convergence acceleration algorithm: QuasiNewton, 3 secant conditions
Point estimation will terminate when objective function delta < 0.0001
Estimation of standard errors disabled; will compute point estimates only.
Size of G: 300x200
Performing five EM steps to prime main algorithm
1 (EM) 	Elapsed: 0.007	Loglikelihood: -77823.3	(delta): 15736.8
2 (EM) 	Elapsed: 0.006	Loglikelihood: -76259.4	(delta): 1563.86
3 (EM) 	Elapsed: 0.006	Loglikelihood: -75633.3	(delta): 626.109
4 (EM) 	Elapsed: 0.006	Loglikelihood: -75198.8	(delta): 434.452
5 (EM) 	Elapsed: 0.006	Loglikelihood: -74815.9	(delta): 382.92
Initial loglikelihood: -74815.9
Starting main algorithm
1 (QN/Block) 	Elapsed: 0.017	Loglikelihood: -60679	(delta): 14136.9
2 (QN/Block) 	Elapsed: 0.017	Loglikelihood: -58624	(delta): 2055.01
3 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -57027.6	(delta): 1596.35
4 (QN/Block) 	Elapsed: 0.021	Loglikelihood: -56685.6	(delta): 342.009
5 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56647.9	(delta): 37.7723
6 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56639.9	(delta): 7.92688
7 (QN/Block) 	Elapsed: 0.02	Loglikelihood: -56638.6	(delta): 1.31102
8 (QN/Block) 	Elapsed: 0.02	Loglikelihood: -56638	(delta): 0.606295
9 (QN/Block) 	Elapsed: 0.021	Loglikelihood: -56637	(delta): 1.05705
10 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56636.2	(delta): 0.812678
11 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56635.2	(delta): 0.917435
12 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56634	(delta): 1.24987
13 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56632.6	(delta): 1.36713
14 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56631.4	(delta): 1.23474
15 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56630	(delta): 1.34865
16 (QN/Block) 	Elapsed: 0.021	Loglikelihood: -56628.6	(delta): 1.46194
17 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56627.1	(delta): 1.45509
18 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56626.4	(delta): 0.760354
19 (QN/Block) 	Elapsed: 0.02	Loglikelihood: -56625	(delta): 1.33425
20 (QN/Block) 	Elapsed: 0.021	Loglikelihood: -56623.9	(delta): 1.08823
21 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56623.3	(delta): 0.661215
22 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56622.6	(delta): 0.645075
23 (QN/Block) 	Elapsed: 0.02	Loglikelihood: -56621.9	(delta): 0.696155
24 (QN/Block) 	Elapsed: 0.023	Loglikelihood: -56621.2	(delta): 0.760023
25 (QN/Block) 	Elapsed: 0.021	Loglikelihood: -56620.5	(delta): 0.69203
26 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56620	(delta): 0.511172
27 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56619.6	(delta): 0.410136
28 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56619.2	(delta): 0.311524
29 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56619	(delta): 0.292117
30 (QN/Block) 	Elapsed: 0.02	Loglikelihood: -56618.8	(delta): 0.17255
31 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56618.7	(delta): 0.106229
32 (QN/Block) 	Elapsed: 0.02	Loglikelihood: -56618.4	(delta): 0.263896
33 (QN/Block) 	Elapsed: 0.02	Loglikelihood: -56618.3	(delta): 0.0890991
34 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56618.2	(delta): 0.112667
35 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56618	(delta): 0.192313
36 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56617.9	(delta): 0.088834
37 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56617.8	(delta): 0.108725
38 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56617.7	(delta): 0.136515
39 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56617.6	(delta): 0.101584
40 (QN/Block) 	Elapsed: 0.02	Loglikelihood: -56617.5	(delta): 0.0424529
41 (QN/Block) 	Elapsed: 0.021	Loglikelihood: -56617.5	(delta): 0.0303441
42 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56617.5	(delta): 0.015413
43 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56617.5	(delta): 0.0311802
44 (QN/Block) 	Elapsed: 0.021	Loglikelihood: -56617.4	(delta): 0.0184339
45 (QN/Block) 	Elapsed: 0.02	Loglikelihood: -56617.4	(delta): 0.0128658
46 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56617.4	(delta): 0.0446392
47 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56617.4	(delta): 0.0197928
48 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56617.3	(delta): 0.0233602
49 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56617.3	(delta): 0.0339324
50 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56617.3	(delta): 0.0509821
51 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56617.2	(delta): 0.0586671
52 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56617.2	(delta): 0.0406663
53 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56617.1	(delta): 0.0399355
54 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56617.1	(delta): 0.0118194
55 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56617.1	(delta): 0.00643737
56 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56617.1	(delta): 0.0215645
57 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56617.1	(delta): 0.0259443
58 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56617	(delta): 0.0316239
59 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56616.9	(delta): 0.100954
60 (QN/Block) 	Elapsed: 0.02	Loglikelihood: -56616.8	(delta): 0.0910732
61 (QN/Block) 	Elapsed: 0.021	Loglikelihood: -56616.7	(delta): 0.132785
62 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56616.5	(delta): 0.173162
63 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56616.2	(delta): 0.321738
64 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56616	(delta): 0.250264
65 (QN/Block) 	Elapsed: 0.021	Loglikelihood: -56615.8	(delta): 0.159833
66 (QN/Block) 	Elapsed: 0.02	Loglikelihood: -56615.7	(delta): 0.0952357
67 (QN/Block) 	Elapsed: 0.021	Loglikelihood: -56615.6	(delta): 0.0536764
68 (QN/Block) 	Elapsed: 0.02	Loglikelihood: -56615.6	(delta): 0.0902296
69 (QN/Block) 	Elapsed: 0.021	Loglikelihood: -56615.5	(delta): 0.0368359
70 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56615.5	(delta): 0.0650332
71 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56615.4	(delta): 0.0806033
72 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56615.3	(delta): 0.111881
73 (QN/Block) 	Elapsed: 0.022	Loglikelihood: -56615.1	(delta): 0.119361
74 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56615.1	(delta): 0.0694357
75 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56615	(delta): 0.03544
76 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56615	(delta): 0.0381645
77 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56615	(delta): 0.015129
78 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56615	(delta): 0.0070295
79 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56615	(delta): 0.00438325
80 (QN/Block) 	Elapsed: 0.023	Loglikelihood: -56615	(delta): 0.000522811
81 (QN/Block) 	Elapsed: 0.019	Loglikelihood: -56615	(delta): 7.85154e-05
Summary:
Converged in 81 iterations (1.798 sec)
Loglikelihood: -56614.968824
Fst divergences between estimated populations:
	Pop0	Pop1	Pop2
Pop0
Pop1	0.374
Pop2	0.366	0.157
Pop3	0.339	0.364	0.373
Writing output files.